e2e/test/scenarios/organization/content-verification.cy.spec.js

```diff
@@ -176,7 +171,7 @@ describeEE("scenarios > premium > content verification", () => {
 
         questionInfoButton().click();
         cy.findByTestId("sidebar-right")
-          .findAllByText(`${adminFullName} verified this`)
+          .findAllByText(`A moderator verified this`)
```

Comment:
Since the endpoint is locked down for non-admins, the name of the verifier can't be grabbed when the user is not an admin, so the more generic message is shown instead. I think this is sensible behaviour, so I just changed this e2e test to search for that string instead.

Reply:
FYI @luizarakaki